dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletResponse.java

```diff
@@ -103,7 +103,7 @@
     // LIBERTY protected WSServletOutputStream _bufferedOut;
     protected ServletOutputStream _bufferedOut;
 
-    protected BufferedWriter _bufferedWriter = new BufferedWriter(DEFAULT_BUFFER_SIZE);
+    protected BufferedWriter _bufferedWriter = new BufferedWriter(0);
```

Comment:
If you are defaulting to 0 in the _bufferWriter, shouldn't the _bufferSize also change to default to 0 as well?

There are other uses for DEFAULT_BUFFER_SIZE in other code.  Does this change to default to 0 and clean to 0 affect that code and require changes there as well?

Reply:
_bufferSize can be default to 0 as well.  I guess that does not make any difference (in term of performance) since _bufferSize will be reset to DEFAULT_BUFFER_SIZE in every request during the initForNextResponse() process.

This change to default 0 and clean do not affect other code.  They are all reset during the initForNextResponse() and getWriter() before writing out any text data.

Reply:
So that things are consistent, I think you should change:

protected int _bufferSize = DEFAULT_BUFFER_SIZE;

to:

protected int _bufferSize = 0;

Reply:
set _bufferSize = 0 during declaration and also in cleanupFromFinish()